shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2272,11 +2290,57 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([method isEqualToString:kUpdateConfigMethod]) {
     [self updateConfig:args];
     result(nil);
+  } else if ([method isEqualToString:kOnInteractiveKeyboardPointerMoveMethod]) {
+    CGFloat pointerY = (CGFloat)[args[@"pointerY"] doubleValue];
+    [self handlePointerMove:pointerY];
+    result(nil);
   } else {
     result(FlutterMethodNotImplemented);
   }
 }
 
+- (void)handlePointerMove:(CGFloat)pointerY {
+  double screenHeight = [UIScreen mainScreen].bounds.size.height;
+  double keyboardHeight = _keyboardRect.size.height;
+  // Determines if pointer goes overtop of the keyboard
+  if (screenHeight - keyboardHeight <= pointerY) {
+    // If no screenshot has been taken
+    if (_keyboardViewContainer.subviews.count == 0) {
+      [self takeScreenshot];
+      [self hideKeyboardWithoutAnimation];
+    } else {
+      CGRect frameRect = _keyboardRect;
+      frameRect.origin.y = pointerY;
+      _keyboardViewContainer.frame = frameRect;
```

Comment:
nits: could you refactor these lines into a method to match what you did in `if`
Maybe something like:
`[self setKeyboardContainerHeight:pointerY]

Reply:
Done! Always helpful to simplify code into sections to increase understandability.